client/performance-profiler/components/metric-tab-bar/style.scss

```diff
@@ -40,9 +40,9 @@ $blueberry-color: #3858e9;
 		outline: 3px solid transparent;
 
 		&.metric-tab-bar__performance {
-			border-inline-width: 0;
+			border-width: 0;
 			margin-inline: -1px;
-			padding-inline: 24px;
+			padding: 17px 24px;
 			width: calc(100% + 2px);
 			box-shadow: $metric-tab-shadow inset;
```

Comment:
These changes were needed because there was a slither of white border poking out the top of the selected performance button. And if you looked super close you would notice that the things weren't perfectly aligned along the top.

![CleanShot 2025-05-13 at 18 43 14@2x](https://github.com/user-attachments/assets/73ab991b-031d-4daf-a1bc-6a19cdf5cedd)

If you are testing that things align perfectly, note that the blue ring around the selected tab _intentionally_ overflows the tab button by `0.5px`. This was part of the original design. A tiny bit of polish to make the selected state look just right.